api/types/types.go

```diff
@@ -9,3 +9,13 @@ type ContainerCreateResponse struct {
 	// Warnings are any warnings encountered during the creation of the container.
 	Warnings []string `json:"Warnings"`
 }
+
+// ContainerExecCreateResponse contains the information returned to a client on the
+// execution of a command in a container.
```

Comment:
Maybe just API `endpoint` for which it used, like `exec/create`(not sure what it is actually)